### ISSUE TITLE
Consistency checks

### ISSUE THREAD

**Issue by nichtich** [gbv/coli-rich#24]
Existing trusted mappings should be analysed for consistency. E.g.

* If there is a mapping α = β or α < β then there must not be a mapping β > α

* If there is a mapping α > β then there must not be a mapping α = β or α < β

* For every mapping α = β or α < β if β (α ≤ β) has no narrower classes then for all transitively narrower classes γ of α there is an implicit mapping γ < β. If a counter-case of a transitively narrower classe γ of α is found where the mapping relation γ < β does not hold, then the mapping α = β oder α < β must be wrong.

In short, the network of KOS hierarchy and mapping relations must be a [partial order](https://en.wikipedia.org/wiki/Partially_ordered_set) (poset) or directed acyclic graph (DAG).
